test/e2e/settings/keep-alive.go

```diff
@@ -74,6 +74,15 @@ var _ = framework.DescribeSetting("keep-alive keep-alive-requests", func() {
 			})
 		})
 
+		ginkgo.It("should set keepalive time to upstream server", func() {
+			f.UpdateNginxConfigMapData("upstream-keepalive-time", "1h")
```

Comment:
Promise, last comment. Dont use the default value to check if this works. 

Set a different value so we can see the configmap reflecting on the config